chain/network/src/types.rs

```diff
@@ -662,6 +666,7 @@ pub enum PeerManagerMessageResponse {
     PeerRequestResult(PeerRequestResult),
     PeersResponseResult(()),
     PeerResponse(PeerResponse),
+    #[cfg(feature = "test_features")]
     GetPeerIdResult(GetPeerIdResult),
```

Comment:
Does `PeerManagerMessageResponse` ever get borsh-serialized (I am not sure what `MessageResponse` derive does)? If so, we cannot skip the field since that would break the protocol due to serialization mismatch between nodes of different versions.

Reply:
@frol Great question! 

`PeerManagerMessageResponse` is just used internally by `PeerManagerActor`, so it's not going to get serialized.
In order to send a message between `Actors` internally.` Actix` requires a type implement `Message`, and it's reply `MessageResponse`. 

`GetPeerId` is the message we send during testing. While `GetPeerIdResult` is just a reply to that `Actix` mesasge.

Btw, I think we should separate our `network protocol` code, see: https://github.com/near/nearcore/issues/5653.
This will make it more clear.